aspnetcore/tutorials/min-web-api.md

```diff
@@ -676,23 +673,23 @@ Copy and paste the **Request URL** in the browser: `https://localhost:<port>/Wea
 
 ## Update the generated code
 
-This tutorial focuses on creating a web API, so we'll delete the Swagger code and the `WeatherForecast` code. Replace the contents of the `Program.cs` file with the following:
```

Comment:
Instead of requiring users to update the generated code, I wonder if it makes more sense to have users start with the smaller web template via:

```
$ dotnet new web -O MinimalApiApp
```

Or the "ASP.NET Core Empty" template in Visual Studio. It'll allow us to remove this content section in the tutorial and help readers get to the main point of the tutorial faster. Thoughts?

Reply:
Makes sense to me. That would also eliminate any confusion over why we start working with a weather forecast API when we say up front that we're going to build a to-do API. The section we're considering demonstrates the Swagger UI -- we could transpose that to the to-do section but it might be better to leave it out since there is already a tutorial dedicated to covering Swagger.

@Rick-Anderson -- what do you think?

Reply:
> @Rick-Anderson -- what do you think?

I agree.